spec/models/case_log_spec.rb

```diff
@@ -764,166 +787,113 @@
       end
     end
 
-    context "Validate pregnancy questions" do
```

Comment:
Are these being deleted or just moved?

Reply:
The whole section was duplicated, it was around line ~100-120